tests/test_wave_03.py

```diff
@@ -43,7 +43,7 @@ def test_friends_unique_movies():
     assert FANTASY_4 in friends_unique_movies
     assert amandas_data == clean_wave_3_data()
 
-@pytest.mark.skip()
+# @pytest.mark.skip()
```

Comment:
Missing assertions here. This test should check that the list contains the correct 3 movies.